corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -181,7 +181,15 @@
 
 - id: persistent_menu
   name: Show Persistent Menu
-  description: Show a persistent menu instead of breadcrumbs.
+  description: Show the persistent menu on the side of the web app.
+  toggle: PERSISTENT_MENU_SETTING
+  widget: bool
+  default: false
+  since: '2.54'
+
+- id: show_breadcrumbs
```

Comment:
What would you think of incorporating `web_apps` into the name? This matters less if the plan is to eventually deprecate this setting.

Reply:
(I'm ambivalent, I see this is part of the new web apps grouping of settings, I'm just thinking about people who are reading code or app files.)